pom.xml

```diff
@@ -105,9 +108,9 @@
                 <version>${bouncycastle.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-engine</artifactId>
-                <version>${junit.version}</version>
+                <groupId>org.junit.platform</groupId>
+                <artifactId>junit-platform-engine</artifactId>
```

Comment:
Can you clarify why this was switched to use the `junit-platform-engine`?  That is an API for writing engines, but I think we want to use the `junit-jupiter-engine` following the example I linked earlier.